python/mp/effects/__init__.py

```diff
@@ -23,5 +23,6 @@
     'looper',
     'wheel',
     'roulette',
-    'random_fill'
+    'random_fill',
+    'trigger_hijack_demo'
 ]
```

Comment:
If you add your new effect in the middle of the list then we won't have conflicts when merging multiple new effects.